Tensile/Source/TensileTypes.h

```diff
@@ -310,23 +310,28 @@ class ProblemKey
 
     // Constructor accepts variable number of sizes:
     template <typename... Ts>
-    ProblemKey(Ts... args)
+    explicit ProblemKey(Ts... args)
+    : _db(0)
     {
         init<NumSizes - 1>(args...);
     }
 
+    
     template <int FirstStride,
               int LastStrideD,
               int LastStrideC,
               int LastStrideA,
               int LastStrideB,
               int NumDimSizes>
+    explicit
     ProblemKey(const ProblemDims<FirstStride,
                                  LastStrideD,
                                  LastStrideC,
                                  LastStrideA,
                                  LastStrideB,
-                                 NumDimSizes>& pdims)
```

Comment:
cppcheck: 
- Class 'ProblemKey' has a constructor with 1 argument that is not explicit.
- Member variable 'ProblemKey::_equalStrides' is not initialized in the constructor.
- Member variable 'ProblemKey::_db' is not initialized in the constructor.